### PR TITLE
Some minor UI improvements for 3.0.1

### PR DESCRIPTION
**Pull request by Nilvus**
After some new feedbacks and tests, just minor improvements to improve (a little bit) contrast on themes. Just a proposal and WIP actually. I will let this PR open to see if some new feedback are coming on different sources and maybe propose new improvements (or not).
Note that first commit have been tested with few users. Some see a difference, others not, so it is also user related (eye view, screen calibration and ambient light). 

Any feedback are, of course, welcomed.

### CODE REVIEW

File: data/themes/darktable.css [hunk 1]
```diff
@@ -483,7 +456,7 @@ entry selection
   padding: 0 2pt 2pt 0.5em;
   font-weight: normal;
   font-family: sans-serif;
-  font-size: larger;
+  font-size: 1.1em;
```
Comment (Nilvus) [inline data/themes/darktable.css]: This one is already set on master like that. Not a changed one, probably about the fact I started this on next branch.

**Comment by TurboGit** [comment]
So this is not WIP anymore? The subject still has WIP so I was waiting.... Can you clarify ? Thanks.

**Comment by Nilvus** [comment]
> So this is not WIP anymore? The subject still has WIP so I was waiting.... Can you clarify ? Thanks.

It's still WIP. Quite long to explain here so I answer on MP in framateam.

**Comment by Nilvus** [comment]
Last commit fix : #4026 ; thanks to elstoc

@TurboGit : from what I have said on framateam, **I consider now this PR good to be merged.** Finally, the 2 first font-sizes are 90% of font text on darktable. The other can't be moved without breaking anything. So it's great as it is by now.

**Comment by TurboGit** [comment]
Manually merged into master and 3.0.1 branch. Thanks!

**Pull request closed without merging**